compiler/src/scala/quoted/runtime/impl/QuotesImpl.scala

```diff
@@ -172,6 +172,30 @@ class QuotesImpl private (using val ctx: Context) extends Quotes, QuoteUnpickler
       end extension
     end ImportMethodsImpl
 
+    type Export = tpd.Export
+
+    object ExportTypeTest extends TypeTest[Tree, Export]:
+      def unapply(x: Tree): Option[Export & x.type] = x match
+        case tree: (tpd.Export & x.type) => Some(tree)
+        case _ => None
+    end ExportTypeTest
+
+    object Export extends ExportModule:
+      def apply(expr: Term, selectors: List[ImportSelector]): Export =
```

Comment:
@nicolasstucki @smarter should the user be able to construct/copy exports, they should only legally appear in a class body

Reply:
Probably not, if you construct an export node in tasty reflect it wouldn't make sense since the typer won't actually add the exported definitions.

Reply:
I've removed apply and copy